.github/workflows/humble-source-build.yml

```diff
@@ -37,7 +37,7 @@ jobs:
             https://raw.githubusercontent.com/ros2/ros2/master/ros2.repos
             https://raw.githubusercontent.com/${{ github.repository }}/${{ github.sha }}/Universal_Robots_ROS2_Driver.${{ env.ROS_DISTRO }}.repos
           colcon-mixin-repository: https://raw.githubusercontent.com/colcon/colcon-mixin-repository/master/index.yaml
-          skip-tests: true
+          extra-cmake-args: -DUR_ROBOT_DRIVER_BUILD_INTEGRATION_TESTS=ON
```

Comment:
```suggestion
```

I would not expect the integration tests to run here, since we are again building inside a docker container. You can mount the docker socket into this container, but [according to the docs](https://docs.github.com/en/actions/using-jobs/running-jobs-in-a-container#setting-container-resource-options) it is not possible to set the network of that container, as you did with the industrial_ci.

In my opinion that's fine to omit the integration tests here, as the source builds aim for more long-term API incompatibilities.

Reply:
I am not quite convinced that this doesn't work. The problem in the (semi-)binary builds was that we had a setup like this:
```
Github container
  |-> ros_industrial container
    |-> ursim
```
In my understanding, mounting the socket and setting the network turned that into this:

```
Github container
  |-> ros_industrial container (network=ursim_net)
  |-> ursim (network=ursim_net)
```
This worked fine. In the scenario here, however, we only have the following:
```
Github container
  |-> ursim (network=ursim_net)
```
with the tests running directly in the github runner. I don't see how this would require any additional configuration to work, and if there is no technical reason to not have these tests run i would prefer to have them for all test configurations.

Reply:
I am not sure whether the deduced structure is correct, as by defining the container an additional docker run step is added in the workflow run. You might as well also be correct though.

Reply:
I temporarily added a commit that runs the source tests on pull requests to check this behavior